plugins/modules/swim_intent.py

```diff
@@ -27,7 +27,6 @@
 extends_documentation_fragment:
   - cisco.dnac.intent_params
 author: Madhan Sankaranarayanan (@madhansansel)
-        Rishita Chowdhary (@rishitachowdhary)
```

Comment:
Please kindly don't remove existing authors..

Reply:
reverted the change